File: WindowsApplication1/Module1.cs

namespace WindowsApplication1
{
    static class Module1
    {
        public static int PropIndex;
        public static int selMatterIndex;
        public static double[,] F = new double[619, 33];
        public static double T;
        public static byte PotencialIndex;

        public static double[] Sigma = new double[101], Epsilon = new double[101], MM = new double[101];

        // _____________MD

        public const int AtomCountmax = 1000000;
        public static byte ConfigurationIndex;
        public static byte DimIndex = 2; // 2D
        public static byte selMatterPotencialIndex;

        public static int AtomCount, AtomCountP, AtomCountC, AtomCountLine, AtomCountAll;
        public static double MinDist; // минимальное расстояние между атомами
        public static double[] X = new double[1000001], Y = new double[1000001], Z = new double[1000001];
        public static double[] Vx = new double[1000001], Vy = new double[1000001], Vz = new double[1000001];
        public static double[] Xk = new double[1000001], Yk = new double[1000001], Zk = new double[1000001];
        public static double[] Vxk = new double[1000001], Vyk = new double[1000001], Vzk = new double[1000001];
        public static double[] Fx = new double[1000001], Fy = new double[1000001], Fz = new double[1000001];
        public static byte[] ConfigP_C = new byte[1000001]; // 
        public static double[] m = new double[1000001];
        public static double Uij, ax, ay, az;
        public static double tau, tau1, dtau; // время
        public static byte GridType;
        public static byte SelectMode; // 1-green,2- orange
        public static bool ViewAtom = true;
        public static bool BreakOn = false;
        public static double Mel = 1d;
        public static int Rows = 1;



        public static string GetConnStr(string sourceName)
        {
            string GetConnStrRet = default;
            GetConnStrRet = "Provider=Microsoft.Jet.OLEDB.4.0;Persist Security Info=True;Data Source=" + sourceName;
            return GetConnStrRet;
            // If InStr(CurDir$, "A") > 0 Then
            // ChDrive Mid(App.path, 1, 1)
            // ChDir App.path
            // End If
            // Provider=Microsoft.Jet.OLEDB.4.0;Data Source=D:\free\Struc.TDN;Persist Security Info=True;Jet OLEDB:Database Password=123
        }
    }
}

File: WindowsApplication1/Module1.vb
Module Module1
    Public PropIndex As Integer
    Public selMatterIndex As Integer
    Public F(618, 32) As Double
    Public T As Double
    Public PotencialIndex As Byte

    Public Sigma(100), Epsilon(100), MM(100) As Double

    '_____________MD

    Public Const AtomCountmax = 1000000
    Public ConfigurationIndex As Byte
    Public DimIndex As Byte = 2 '2D
    Public selMatterPotencialIndex As Byte

    Public AtomCount, AtomCountP, AtomCountC, AtomCountLine, AtomCountAll As Integer
    Public MinDist As Double 'минимальное расстояние между атомами
    Public X(AtomCountmax), Y(AtomCountmax), Z(AtomCountmax) As Double
    Public Vx(AtomCountMax), Vy(AtomCountMax), Vz(AtomCountMax) As Double
    Public Xk(AtomCountMax), Yk(AtomCountMax), Zk(AtomCountMax) As Double
    Public Vxk(AtomCountMax), Vyk(AtomCountMax), Vzk(AtomCountMax) As Double
    Public Fx(AtomCountmax), Fy(AtomCountmax), Fz(AtomCountmax) As Double
    Public ConfigP_C(AtomCountmax) As Byte '
    Public m(AtomCountmax) As Double
    Public Uij, ax, ay, az As Double
    Public tau, tau1, dtau As Double 'время
    Public GridType As Byte
    Public SelectMode As Byte '1-green,2- orange
    Public ViewAtom As Boolean = True
    Public BreakOn As Boolean = False
    Public Mel As Double = 1
    Public Rows As Integer = 1



    Public Function GetConnStr(ByVal sourceName As String) As String
        GetConnStr = "Provider=Microsoft.Jet.OLEDB.4.0;Persist Security Info=True;Data Source=" & sourceName
        '        If InStr(CurDir$, "A") > 0 Then
        '            ChDrive Mid(App.path, 1, 1)
        '            ChDir App.path
        '        End If
        'Provider=Microsoft.Jet.OLEDB.4.0;Data Source=D:\free\Struc.TDN;Persist Security Info=True;Jet OLEDB:Database Password=123
    End Function
End Module





